### PR TITLE
chore: add explanations about how to use gofumpt in vim

### PR DESCRIPTION
**Pull request by grepsuzette**
# How has this been tested?

It's what I have been using for a ~week~month or two.
There may be more succint ways to do this.

BTW if someone knows how to have lint enabled in vim on .gno files, let me know.

### CODE REVIEW

**Comment by tbruyelle** [comment]
Have you seen this https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md#vim-support ?
This is what I use and it works well.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ augroup gno_autocmd
 augroup END
 ```
 
+To use *gofumpt* instead of *gofmt*, as hinted in the comment, you may substitute the cexpr line above with the following (please make sure to replace `<path/to/gno>` with the path to your local gno repository):
+
+`cexpr system('go run -modfile </path/to/gno>/misc/devdeps/go.mod mvdan.cc/gofumpt -w ' . expand('%'))`
```
Comment (thehowl) [inline CONTRIBUTING.md:95]: Change this to be a codeblock with \`\`\`vim?

**Review by thehowl — COMMENTED** [review]
For me I just changed `gofmt` to `gofumpt` as I had it installed in my `$PATH` :smile: But this can be a good way to avoid installing it.

For what regards linting in Gno files, I haven't configured it yet. But it probably can be done in a "basic" manner with [revive](https://github.com/mgechev/revive#text-editors). It would be cool to have it with golangci-lint using the configuration of the repository and adding support for linting gno files.

**Comment by tbruyelle** [comment]
> Have you seen this https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md#vim-support ? This is what I use and it works well.

Oh sorry I didn't notice this was a PR so I hadn't look at the code before submitting my answer :sweat_smile:

**Review by tbruyelle — APPROVED** [review]

**Review by thehowl — APPROVED** [review]

**Pull request merged**